src/firebase_functions/private/util.py

```diff
@@ -243,3 +243,44 @@ def on_call_check_tokens(request: _Request,) -> _OnCallTokenVerification:
                          log_payload)
 
     return verifications
+
+
+@_dataclasses.dataclass(frozen=True)
+class FirebaseConfig():
+    """
+    A collection of configuration options needed to
+    initialize a firebase App.
+    """
+
+    storage_bucket: _typing.Optional[str]
+    """
+    The name of the Google Cloud Storage bucket used for storing application data.
+    This is the bucket name without any prefixes or additions (without "gs://").
+    """
+
+    # TODO more to be added later when they are required
```

Comment:
@taeold FYI - left this as a TODO since none of the others were needed yet - thought it best to add them when they're required - is that ok?

Reply:
I'd maybe add databaseURL here, but fair with me if you want to tackle it later.